src/Analysis/Ast/Impl/Specializations/Typing/TypingModule.cs

```diff
@@ -49,6 +50,14 @@ internal sealed class TypingModule : SpecializedModule {
             // TypeVar
             var fn = PythonFunctionType.Specialize("TypeVar", this, GetMemberDocumentation("TypeVar"));
             var o = new PythonFunctionOverload(fn.Name, location);
+            o.SetParameters(new List<ParameterInfo> {
+                    new ParameterInfo("name", Interpreter.GetBuiltinType(BuiltinTypeId.Str), ParameterKind.Normal, null),
+                    new ParameterInfo("*constraints", Interpreter.GetBuiltinType(BuiltinTypeId.Str), ParameterKind.List, null),
```

Comment:
Should the names include the `*`, or should that be dependent on `ParameterKind.List`? I'd write a function with `*args` and see what it's represented as first.

Reply:
Yeah the ```*``` shouldn't be there, I just tested it.

Reply:
You are right, I wrote a test and it should be constraints.